lesson09/index.js

```diff
@@ -0,0 +1,51 @@
+const ul = document.getElementById("js-lists");
+
+const attributes = [
+    { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+    { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+];
+
+function addLoading() {
+    const li = document.createElement("li");
+    const img = document.createElement("img");
+    li.id = "js-loading";
+    img.src = "loading-circle.gif";
+
+    ul.appendChild(li).appendChild(img);
+};
+
+function removeLoading() {
+    const li = document.getElementById("js-loading");
+    ul.removeChild(li);
+}
+
+function fetchListData() {
+    addLoading();
+    return promise = new Promise(resolve => {
```

Comment:
```suggestion
    return new Promise(resolve => {
```
IMO
I feel it is common to write it this way.
I also felt that the "promise" in "return promise" was unnecessary, as it didn't seem to have any particular role.